build/secondary/third_party/libpng/BUILD.gn

```diff
@@ -0,0 +1,51 @@
+# Copyright 2014 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+config("libpng_public") {
+  include_dirs = [ "." ]
+}
+
+source_set("libpng") {
+  public_configs = [ ":libpng_public" ]
+
+  cflags_c = []
+  defines = []
+
+  sources = [
+    "png.c",
```

Comment:
This might need to be `//flutter/third_party/libpng/png.c`

Reply:
> `ninja: error: '../../flutter/third_party/libpng/png.c', needed by 'obj/flutter/third_party/libpng/libpng.png.o', missing and no known rule to make it`